Introduction.Rmd

```diff
@@ -126,14 +126,57 @@ This is a lot to learn, but don't feel overwhelmed. Start with a minimal subset
+
+  * devtools: HW, [Winston Chang][winston], [Jim Hester][jim] (maintainer, >= v1.13.5)
+  * usethis: HW, [Jennifer Bryan][jenny] (maintainer >= v1.5.0)
+  * testthat: HW (maintainer)
+  * roxygen2: HW (maintainer), [Peter Danenburg][klutometis], [Manuel Eugster][mjaeugster]
+  * remotes: HW, [Jim Hester][jim] (maintainer), [Gábor Csárdi][gabor], [Winston Chang][winston], [Martin Morgan][mtmorgan], [Dan Tenenbaum][dtenenba]
+  * desc: [Gábor Csárdi][gabor] (maintainer), [Kirill Müller][kirill], [Jim Hester][jim]
+  * pkgbuild: HW, [Jim Hester][jim] (maintainer)
+  * pkgload: HW, [Jim Hester][jim] (maintainer), [Winston Chang][winston]
+  * rcmdcheck: [Gábor Csárdi][gabor] (maintainer)
+  * revdepcheck: HW, [Gábor Csárdi][gabor] (maintainer)
+  * sessioninfo: HW, [Gábor Csárdi][gabor] (maintainer), [Winston Chang][winston], [Robert Flight][rmflight], [Kirill Müller][kirill], [Jim Hester][jim]
+  * withr: HW, [Jim Hester][jim] (maintainer), [Kirill Müller][kirill], [Kevin Ushey][kevin], [Winston Chang][winston], [Richard Cotton][richierocks]
+
+This book and the R package development community benefit tremendously from experts who smooth over specific pain points:
 
-The tools in this book wouldn't be possible without many open source contributors. [Winston Chang](https://github.com/wch/), my co-author on devtools, spent hours debugging painful S4 and compiled code problems so that devtools can quickly reload code for the vast majority of packages. [Kirill Müller](https://github.com/krlmlr) contributed great patches to many of my package development packages including devtools, testthat, and roxygen2. [Kevin Ushey](http://github.com/kevinushey), [JJ Allaire](https://github.com/jjallaire) and [Dirk Eddelbuettel](http://dirk.eddelbuettel.com) tirelessly answered all my basic C, C++ and Rcpp questions. [Peter Danenburg](https://github.com/klutometis) and [Manuel Eugster](http://www.statistik.lmu.de/~eugster/) wrote the first version of roxygen2 during a Google Summer of Code. [Craig Citro](https://github.com/craigcitro) wrote much of the code to allow travis to work with R packages.
+  * [Kevin Ushey][kevin], [JJ Allaire](https://github.com/jjallaire), and [Dirk Eddelbuettel](http://dirk.eddelbuettel.com) tirelessly answered all sorts of C, C++, and Rcpp questions.
+  * [Craig Citro](https://github.com/craigcitro) wrote much of the initial code to facilitate using Travis-CI with R packages.
+  * [Jeroen Ooms](https://github.com/jeroen) also helps to maintain R community infrastructure, such as the current R support for Travis-CI (along with Jim Hester), and the Windows toolchain.
```

Comment:
I reworked this in light of:

  * The new section just above, which is package-driven. Here I mostly tried to retain mentions of people that don't come up via the packages. Kevin is the only repeat.
  * Recent reality in the Travis / Windows tooling landscape.